### DISCUSSION TITLE
Can't update the TeslaMate on docker. No such file or directory

### DISCUSSION THREAD

**Discussion by fatrnick** [teslamate-org/teslamate#3590] [General]
When I SHH the docker and go the teslamate, it's not found. But it exists on the docker folder. 
Anyone can help?

<img width="480" alt="CleanShot 2023-12-28 at 00 28 44@2x" src="https://github.com/teslamate-org/teslamate/assets/14029112/76ba5155-c738-4c24-94d9-a8ada570e8d1">
<img width="338" alt="CleanShot 2023-12-28 at 00 33 12@2x" src="https://github.com/teslamate-org/teslamate/assets/14029112/74d62aad-b959-4273-a5fa-23404d13b8b2">

<img width="595" alt="CleanShot 2023-12-28 at 00 29 26@2x" src="https://github.com/teslamate-org/teslamate/assets/14029112/43b008f0-ab12-41d8-a092-520919e68739">

**Comment by cwanja** [comment]
You are trying `teslamate`.  It is named `TeslaMate`. Case matters in CLIs. Try tab complete.

**Comment by fatrnick** [comment]
It's not working too.😭

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Reply by cwanja** [reply]
It does not look like that directory is in `/etc`. In your `ls`, there is no other files similar to your File Manager screenshot. 

Try using `find` with a directory flag for TeslaMate and see what that returns.  https://www.howtogeek.com/771399/how-to-use-the-find-command-in-linux/#the-linux-find-command
